packages/cli/src/routeGeneration/templates/koa.hbs

```diff
@@ -219,6 +229,15 @@ export function RegisterRoutes(router: KoaRouter) {
                 return validationService.ValidateParam(args[key], context.request.body, name, errorFields, undefined, {{{json minimalSwaggerConfig}}});
             case 'body-prop':
                 return validationService.ValidateParam(args[key], context.request.body[name], name, errorFields, 'body.', {{{json minimalSwaggerConfig}}});
+            case 'formData':
+                if (args[key].dataType === 'file') {
+                  return validationService.ValidateParam(args[key], context.request.file, name, errorFields, undefined, {{{json minimalSwaggerConfig}}});
+                } else if (args[key].dataType === 'array' && args[key].array.dataType === 'file') {
+                  return validationService.ValidateParam(args[key], context.request.files, name, errorFields, undefined, {{{json minimalSwaggerConfig}}});
+                } else {
+                  // When https://github.com/koa-modules/multer/pull/15 gets merged in, the conditional can be removed
```

Comment:
Should we use `@koajs/multer` instead to avoid this?

Reply:
Tbh i took that comment from https://github.com/lukeautry/tsoa/pull/96 and didnt give it any consideration.

Let me look into that and report back later

Reply:
The package [koa/multer](https://www.npmjs.com/package/@koa/multer) points to the github repository [koajs/multer](https://github.com/koajs/multer) meaning that the comment and the conditional is outdated.

This means that we already use the `@koajs/multer` package with the referenced PR already merged. 

I committed the necessary changes

Reply:
@WoH is this resolved?

Reply:
Sorry, I didn't get/missed the notification.
Doesn't that mean we can skip the else entirely?